os/mounting-storage.md

```diff
@@ -35,11 +37,13 @@ coreos:
         Description=Formats the ephemeral drive
         After=dev-xvdb.device
         Requires=dev-xvdb.device
+        ConditionPathExists=!/var/lib/format-xvdb
```

Comment:
Just some additional info: I'm running CoreOS on top of Openstack. When creating a new instance, it's afaik not possible to add a storage volume (which is independent of the instance lifetime) right away. So you can only add a volume after the instance has already booted. That's why I stated that in my case, ConditionFirstBoot didn't work for me. Of course the volume could be formatted outside and just attached.
My main concern related to this issue is that the current Unit file might lead to data loss due to re-formatting drives in use.